### ISSUE TITLE
Incompatible function declaration

### ISSUE THREAD

**Issue by mnabialek** [Codeception/Codeception#2349]
When running in PHP 5.6 I'm getting at the moment:
```
 [ErrorException] Declaration of Codeception\Step\Comment::getHtml() should be compatible with Codeception\Step::getHtml($highlightColor = '#732E81')```

**Comment by vladprokopchuk** [comment]
same errors

**Comment by Naktibalda** [comment]
Which version of Codeception is affected?

**Comment by Naktibalda** [comment]
It was fixed in 2.1.1 https://github.com/Codeception/Codeception/commit/92cf6d17dd75d079ef61e4e3fc44fc7b141d96da

**Comment by vladprokopchuk** [comment]
thx, works
